Decisions.Microsoft365.Exchange/GraphRest.cs

```diff
@@ -90,6 +89,12 @@ public static HttpResponseMessage Delete(ExchangeSettings? settingsOverride, str
                 throw new BusinessRuleException("The request was unsuccessful.", ex);
             }
         }
+        
+        private static HttpResponseMessage CreateHttpRequest(ExchangeSettings? settingsOverride, string urlExtension,
+            HttpContent? content, HttpMethod httpMethod)
+        {
+            return SendHttpRequest(settingsOverride, $"{GetSettings(settingsOverride).GraphUrl}{urlExtension}", content, httpMethod);
+        }
```

Comment:
Why the need for a new method?

Reply:
SendHttpRequest takes a URL input that is fully configured by that point: so it can accept the BaseURL+Extension that each step sends while also allowing the OData URLs to request data.

Reply:
See my comment above.